src/util/awsAccountInfo.js

```diff
@@ -1,8 +1,14 @@
 const fs = require("fs");
 const configDir = require("./configDir.js");
+const hiddenAccountFilePath = `${configDir}"aws_account_info.json"`;
 
```

Comment:
`hiddenAccountFilePath` should be <code>\`${configDir}/aws_account_info.json\`</code>

Reply:
On my machine it expands to `/home/john/.maestro"aws_account_info.json"`
when it should be `/home/john/.maestro/aws_account_info.json`.

Reply:
Was just about to post the same thing.

Reply:
Or it could be `configDir + "/aws_account_info.json"`; doesn't matter.

Reply:
Fixed